### DISCUSSION TITLE
Nemo - "Apply" and "OK" buttons on the properties window

### DISCUSSION THREAD

**Discussion by HT-7** [linuxmint/discussions#118] [Ideas]
Currently, a file or folder name can be changed accidentally because the name field is pre-selected imediately after the "properties" window is opened.

When pressing a key after that, the file name is replaced with that keystroke and it is saved the moment the properties window is closed. Needless to say, this leads to unwanted renames.

Please add "Apply" and "OK" and "Cancel" buttons to avoid this annoyance.

I know, there is an "undo" feature (CTRL+Z), but if a folder is renamed while it is being accessed, such as downloads or file transfers, an existing download or file transfer could be interrupted. This needs to be prevented.

**Comment by Secret-chest** [comment]
Please don't. This is Windows-like and not welcomed on Linux, and it makes it much harder to edit.
